src/NUnitFramework/tests/nunit.framework.tests.csproj

```diff
@@ -17,8 +17,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net35' and '$(TargetFramework)' != 'net40'">
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
```

Comment:
All later packages cause our NUnitLite tests to fail on netcoreapp2.0 and I'm not ready to dig into that.

Reply:
We should probably create an issue for this either here or in https://github.com/microsoft/vstest)

Reply:
Opened https://github.com/nunit/nunit/issues/3327 (sorry that's all I can do right now)

Reply:
Opened https://github.com/nunit/nunit/issues/3327 (sorry that's all I can do right now)